onnx_tf/frontend.py

```diff
@@ -152,20 +168,25 @@ def tensorflow_graph_to_onnx_graph(cls, graph_def, output, name="graph"):
 
         # Check if specialized handler exists.
         if handler_name in dir(cls):
+          input_node = get_nodes_by_name(graph_def.node, node.inputs)
           method_to_call = getattr(cls, handler_name)
-          ops_proto.append(method_to_call(node, consts))
+          ops_proto.append(method_to_call(node, consts, input_node))
```

Comment:
The whole point of having a input_node param seems to be that you want to access the shapes of input nodes, which might be better suited as an attribute in the TensorflowNode class? What do you think? This way we can keep the signature of handler clean (important because this signature is copied in every handler...) as most handlers do not need this param.

Reply:
Yes, I agree with you for the signature. It should be betted to keep it without the input node param.
But I did it because I didn't get the idea to add the input shape to the TensorflowNode class.

I will make the fix, thanks